### PR TITLE
Muffins

### PR DESCRIPTION
**Pull request by RumiTiger**
## About the PR
I added a chocolate and banana muffin to the game. I also resprited the cherry and berry muffins.
I added recipes for almost all the muffins.
I added muffin tins to the game, they are used in muffin recipes. You can print them on the autolathe.

## Why / Balance
I really don't understand why there were muffins in the game, but no way to make them.

## Media
Muffins
{A2D855CE-3C70-4A52-879F-0DA6D98C7CB4}
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
**Changelog**
:cl:
- add: A chocolate and banana muffin has been added to the game. The berry and cherry muffins have also been resprited!
- tweak: Now you can make a regular, chocolate, banana, and berry muffin!
- add: Muffin tins have been added to the game. They are needed for making muffins!

### CODE REVIEW

**Comment by TheShuEd** [comment]
webedit ops is real

**Comment by slarticodefast** [comment]
The test fail seems to be real. One of the problem is that the meta.json has the state cherry-muffin twice. If you get it fixed I can have a look at the PR.

File: Resources/Prototypes/Recipes/Cooking/meal_recipes.yml
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Cooking/meal_recipes.yml:1795]: You can uncomment this now that cherries are merged :)

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/dinnerware.yml
Comment (slarticodefast) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/dinnerware.yml:29]: Put it below the FoodPlateTin. The objects show up in the machine in the order they are defined here, so they should be ordered by category.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml:151]:
```suggestion
```
Remove the banana tag. That one is used for a cargo bounty and we don't want the muffins to fullfill it.

**Review by slarticodefast — CHANGES_REQUESTED** [review]
Ok, tested it in-game and everything works great. Found two more minor things that need to be adjusted. I will ask the art directors for approval of the sprites next.

**Review by UbaserB — CHANGES_REQUESTED** [review]
Could you make the cherry and chocolate cupcakes match the shading of the normal muffin, (with minor hue-shifting not just pure black/white) alongside making them more desaturated? If you wanted to, you could alternatively take aspects from the original cherry muffin sprite and put it onto the normal muffin for consistency.

**Comment by RumiTiger** [comment]
> Could you make the cherry and chocolate cupcakes match the shading of the normal muffin, (with minor hue-shifting not just pure black/white) alongside making them more desaturated? If you wanted to, you could alternatively take aspects from the original cherry muffin sprite and put it onto the normal muffin for consistency.

Like this?
Muffins

**Comment by slarticodefast** [comment]
>Also in the future, please avoid web edits so we can make sure all changes have been properly tested.

Like I asked before, please avoid webedits and push changes to your remote branch using git instead for the following reasons:
- webedits are more susceptible to errors since you may forget to upload a file or submit a wrong one
- we cannot ensure the changes have been locally tested by the author like they should be
- each single commit causes the integration tests to be re-run and there is a total computation limit

Please refer to our documentation to learn how to use git and an IDE:
[Git for the SS14 Developer](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html)
[Setting up a Development Environment](https://docs.spacestation14.com/en/general-development/setup/setting-up-a-development-environment.html)

>Like this?

I will ask the art directors for approval.

File: Resources/Prototypes/Recipes/Lathes/cooking.yml
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/cooking.yml:113]: Move this above FoodKebabSkewer so it is below the FoodPlateTin recipe.
You will also need to add this to lathe.yml so the recipe can be used.
Reminder: No web edits, please test your changes in-game.

**Review by slarticodefast — APPROVED** [review]
Looks good to me.
The sprites have been art director approved.

**Comment by deathride58** [comment]
Can confirm that the reviewing art director has given their approval, but did not state such in the PR. Dismissing their requested changes and merging

**Review by deathride58 — APPROVED** [review]

**Pull request merged**
